### ISSUE TITLE
Catboost.caret not working

### ISSUE THREAD

**Issue by Mariaridel** [catboost/catboost#1138]
Problem: I've been trying to use Catboost.caret to train a model in Rstudio, and whenever I try the session crashes. At the end, I have simplied the code at its most to try to find the problem, but I still get an error that I couldn't solve. I would appreciate any suggestion as I'm really stuck.
catboost version: v0.20

My code:
`catboost.samp <- caret::train(x = dfTrainx, y = as.factor(make.names(dfTrain[,"y_train"])),` 
`                              method=catboost.caret,`
`                              metric='ROC', `
`                             importance = 'impurity',`
`                              trControl = trainControl(method="none", classProbs=TRUE),`
`                              tuneGrid = expand.grid(`
`                                depth=2,`
`                                learning_rate=0.1,`
`                                l2_leaf_reg=3.5,`
`                                rsm=0.75, #% of features to use at each split selection`
`                                border_count= 10, #Number of splits for numerical features`
`                                iterations=100 ),`
`                              replace = TRUE,`
`                              classification = TRUE)`

The error:

**Error in catboost.train(pool, test_pool, param) : 
  c:/goagent/pipelines/buildmaster/catboost.gittt/catboost/private/libs/options/plain_options_helper.cpp:469: Unknown option {replace} with value "true"
Timing stopped at: 1.59 0.07 1.66**

**Comment by pallto** [comment]
I have the same issue. Did you solve it? I'm new here, sorry if there is another way to follow up open issues. Thanks!

**Comment by kizill** [comment]
Hello! Why are you setting `replace`, `importance` and `classification` options for CatBoost? There is no need in them - if you pass factor as label vector caret will automatically switch to `Logloss`. So I guess, you can just drop `replace` and `classification` options, unknown to CatBoost and then everything will work just fine.
So closing as solved for now, please reopen and ask questions if any :)
